src/analysis/retail/mage/arcane/guide/ArcaneBarrage.tsx

```diff
@@ -68,30 +75,35 @@ class ArcaneBarrageGuide extends Analyzer {
         });
       }
 
-      if (ab.arcaneSoul) {
-        tooltipItems.push({ perf: QualitativePerformance.Good, detail: `Had Arcane Soul` });
+      if(!ab.netherPrecisionStacks && ab.clearcasting){
+        tooltipItems.push({
+          perf: QualitativePerformance.Ok,
+          detail: `No Netherprecision stacks even though clearcasting was available `,
+        });
       }
 
-      if (ab.gloriousIncandescence) {
+            if(ab.netherPrecisionStacks ){
         tooltipItems.push({
           perf: QualitativePerformance.Good,
-          detail: `Had Glorious Incandescence`,
+          detail: `Had Netherprecision`,
         });
       }
 
-      if (ab.intuition) {
-        tooltipItems.push({ perf: QualitativePerformance.Good, detail: `Had Intuition` });
+      if (ab.arcaneSoul) {
+        tooltipItems.push({ perf: QualitativePerformance.Good, detail: `Had Arcane Soul` });
       }
 
-      const aethervisionStacks =
-        ab.aethervision && ab.aethervision.stacks >= AETHERVISION_STACK_THRESHOLD;
-      if (aethervisionStacks) {
+      if ((ab.gloriousIncandescence || (ab.burdenOfPower && ab.blastPrecast)) && !touchSoon) {
         tooltipItems.push({
           perf: QualitativePerformance.Good,
-          detail: `Had ${AETHERVISION_STACK_THRESHOLD} Stacks of Aethervision`,
+          detail: `Had Glorious Incandescence`,
```

Comment:
the description here doesn't match the condition. it is possible to have "Had Glorious Incandescence" show up as a reason without actually having Glorious Incandescence with this condition.